types/api.yaml

```diff
@@ -81,3 +81,8 @@ Committee:
       description: "List of validator indices assigned to this committee"
       items:
         $ref: './primitive.yaml#/Uint64'
+
+BroadcastValidation:
+  description: Level of validation that must be applied to a block before it is broadcast.
```

Comment:
I agree in principal, but the problem is as a query parameter driving this, 'must' is hard to achieve. Every client will have to implement this in order go get that outcome...

If a given client doesn't implement the query parameter, it's likely to be silently ignored and default current behaviour happen, and the person setting the query parameter believes it 'must' happen...

It would only take 1 client not implementing the query parameter for this statement to not hold true... I guess if clients are aware of this they can at least throw 'not implemented' or something, but that does require a code change, so this is basically a 'breaking' change we're suggesting...

Reply:
resolved by v2